TRANSCRIPTION.md

```diff
@@ -70,32 +70,25 @@
 | ĩː | 5 | ✓ | ✓ |
 | ʊ̃ | 5 | ✓ | ✓ |
 | j̃ʰ | 4 | ✓ | ✓ |
-| áː | 4 | ✓ | ✓ |
+| áː/aː | 4 | ✓ | ✓ |
 | dʒ | 3 | ✓ | ✓ |
 | ð | 3 | ✓ | ✓ |
 | õ | 3 | ✓ | ✓ |
 | ɾʰ | 3 | ✓ | ✓ |
-| aʰ | 2 | ? | ? |
 | aˀ | 2 | ✓ | ✓ |
 | hʲ | 2 | ✓ | ✓ |
 | z | 2 | ✓ | ✓ |
 | ãː | 2 | ✓ | ✓ |
 | ə̃ | 2 | ✓ | ✓ |
-| iʰ | 1 | ? | ? |
 | i̥ | 1 | ✓ | ✓ |
```

Comment:
Voiceless vowels are described as phonemic in a number of South American languages. An even larger number of people disagree about their existence or the actual phonetic value (I remember some people describing Ancient Greek rough spirits as voiceless vowels, for example), but even though it is a single instance, it is likely what the author intended here. It is not a glide because it is part of a "-ki̥ " suffix, it might be phonetically an aspiration that originated from a proto-vowel (some potential cognates have an /a/ in the alignment).

I would have no problem in mapping to "i̥ /i", however.

Reply:
Just to confirm so we can merge, would it be ok to fix it to `"i̥ /i"`?

Reply:
yes